### PR TITLE
Alphabetize contributors

### PR DESCRIPTION
**Pull request by ben-albrecht**
sleepSorted™ the contributors list.

I've employed the strategy of "sort by the first last name" for people with multiple last names, e.g. "Thomas Van Doren" is sorted by "Van", and "Marcos Cleison Silva Santana" is sorted by "Cleison".

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Contributors to previous releases
 * Joel Denny, [ETI]
 * Roxana Diaconescu, [CalTech]
 * James Dinan, [Cray Inc.] / [The Ohio State University]
+* Thomas Van Doren, [Cray Inc.]
```
Comment (lydia-duncan) [inline CONTRIBUTORS.md]: I don't think this movement is right.  Thomas' last name is "Van Doren", Van isn't his middle name

**Comment by ben-albrecht** [comment]
It does feel wrong...  I asked the internet and the conclusion seems to be
that there are strict rules for sorting multi-word last names, but they
vary from country to country. It's ultimately more practical to pick a
simple set of rules such as by the very last name, and stick to that. We
could alternatively pick a different set of rules, such as sorting by the
first last name.

Given the minimal importance here, I could be swayed easily if one felt
strongly about our approach. :)

**Comment by lydia-duncan** [comment]
I'd be in favor of sorting by first last name :)

**Comment by ben-albrecht** [comment]
>> Given the minimal importance here, I could be swayed easily if one felt
strongly about our approach. :)

> I'd be in favor of sorting by first last name :)

Consider me swayed. I'm sure @thomasvandoren will appreciate it anyway.

**Pull request closed without merging**